01/app.js

```diff
@@ -0,0 +1,12 @@
+const btn = document.querySelector('button');
+let counter = 0;
+const clickCounter = function(){
+
+    if(btn) {
```

Comment:
`if` powinien być jeszcze przed `btn.addEvent...` bo jeśli `btn` będzie `null` (nie zostanie wyszukany) to w tym miejscu będzie błąd.